packages/elements/src/account/types.ts

```diff
@@ -0,0 +1,3 @@
+import { type UserProfileResponse } from '@logto/schemas';
+
+export type UserProfile = Partial<UserProfileResponse>;
```

Comment:
nit: Why partial? Maybe we can use pick instead?

Reply:
The backend returns partial object.